<launch>
    <arg name="config_path" default = "$(find rtkrcv)/config/rtkrcv.conf" />
    <arg name="output_path" default = "$(find rtkrcv)/output/" />
    <node pkg="rtkrcv" name="rtkrcv"  type="rtkrcv-node" output="screen">
        <param name="config_file" type="string" value="$(arg config_path)" />
        <param name="output_dir" type="string" value="$(arg output_path)" />
    </node>
</launch>
